google/ads/google_ads/config.py

```diff
@@ -129,6 +129,14 @@ def load_from_yaml_file(path=None):
 
     return parse_yaml_document_to_dict(yaml_doc)
 
+@_config_validation_decorator
+@_config_parser_decorator
+def load_from_dict(config_dict):
```

Comment:
Add a PyDoc that describes what this function does, what it's parameters are, and what its return value is.